Assets/Runtime/GameEvents/Game/GameEvent.cs

```diff
@@ -39,7 +40,17 @@ public void RaiseGameEvent()
                     Debug.Log($"Raise event: {name}, listener: {listener}");
                 }
 
-                listener.RaiseGameEvent();
+                try
+                {
+                    listener.RaiseGameEvent();
+                }
+                catch (Exception e)
+                {
+                    if (debug)
+                    {
+                        Debug.Log($"Listener: {listener} of event: {name} has thrown an exception: {e.Message}");
```

Comment:
I'd recommend to use `Debug.LogException(Exception exception, Object context)` (https://docs.unity3d.com/ScriptReference/Debug.LogException.html) as not to consume the stack trace.

Reply:
Added `Debug.LogException(Exception exception, Object context)`, but I also left the log informing which listener threw an exception since I think it is very useful for debugging.